sections/product-recommendations.liquid

```diff
@@ -21,10 +21,10 @@
 {%- endstyle -%}
 
 <div class="color-{{ section.settings.color_scheme }} gradient">
-  <product-recommendations class="product-recommendations page-width section-{{ section.id }}-padding isolate" data-url="{{ routes.product_recommendations_url }}?section_id={{ section.id }}&product_id={{ product.id }}&limit=4">
+  <product-recommendations class="product-recommendations page-width section-{{ section.id }}-padding isolate" data-url="{{ routes.product_recommendations_url }}?section_id={{ section.id }}&product_id={{ product.id }}&limit=5">
```

Comment:
I originally suggested this, but @wiktoriaswiecicka made a good point to keep it consistent: https://github.com/Shopify/dawn/pull/989#discussion_r792758554

Maybe this can be an exception and be kept at 4? And only allow the column range to be 1-4

Reply:
Aligned 👍